channeldb/reports.go

```diff
@@ -84,6 +84,11 @@ const (
 	// ResolverOutcomeTimeout indicates that a contract was timed out on
 	// chain.
 	ResolverOutcomeTimeout ResolverOutcome = 4
+
+	// ResolverOutcomeFirstStage indicates that a htlc had to be claimed
+	// over two stages, with this outcome representing the confirmation
+	// of our success/timeout tx.
+	ResolverOutcomeFirstStage ResolverOutcome = 5
```

Comment:
So there will be two `ResolverTypeIncomingHtlc` entries in the `resolversBucket`. One with outcome `ResolverOutcomeFirstStage` and another one with `ResolverOutcomeClaimed`?